Changes to:

Neakas opened the pull request:
- Coloring of Items (Standart-Vam Items are Colored Green)
- Added Basic Handler for Local Settings. The Selected Folderpath and the new Option to Auto-Apply Changes are Saved
- Auto-Apply Changes on D&D to Save on Clicks.

Commit:
Changes to:
- Coloring of Items (Standart-Vam Items are Colored Green)
- Added Basic Handler for Local Settings. The Selected Folderpath and the new Option to Auto-Apply Changes are Saved
- Auto-Apply Changes on D&D to Save on Clicks.

morph1sm commented on Morphology/Regions.cs:
```diff
@@ -66,6 +69,10 @@ public string Folder
+
+            //Apply Green Color for VamStandartRegions to make them Better Visible
+            //Could be moved to ApplyVaMStandardRegions Region Constructors. This is just Lazy
+            this.ToList().ForEach(x => x.DisplayColor = Brushes.Green);
```
@Neakas Let's make this a convention in the Region constructor. When a region is instantiated as a standard region, then the brush is gray, otherwise it is light blue.

I want to mimic the VaM styling of custom morphs (blue) vs built-in (gray). Thanks!

Commit:
Changes to Color-Scheme as commented by morph1sm

Commit:
- Improved Drag and Drop Behaviour
- Created Visual Feedback for the Dragged Items
- Fixed Drag and Drop Bugs
- Improved Visual Representation of Selected Morphs (reported by User DJ)
- Fixed Path Display Bug on to Small Window (reported by User DJ)
- Fixed Items Selecting Bug in conjunction with Pressing the Ctrl or Shift Key (reported by User DJ)

Commit:
- Code Cleanup

morph1sm commented:
Thanks @Neakas !
